cmd/check_container.go

```diff
@@ -168,17 +170,40 @@ func getContainerPolicyExceptions(ctx context.Context, pc pyxisClient) (policy.P
 
 func checkContainerPositionalArgs(cmd *cobra.Command, args []string) error {
 	if len(args) != 1 {
-		return fmt.Errorf("a container image positional argument is required")
+		return fmt.Errorf("a container image positional argument is required or a flag is not specified as --flag=value")
 	}
 
+	cmd.Flags().VisitAll(func(f *pflag.Flag) {
+		if f.Changed && f.Value.String() == "--submit" {
+			// We have --submit in one of the flags. That's a problem.
+			// We will set the submit flag to true so that the next block functions properly
+			submit = true
+		}
+	})
+
+	// --submit was specified
 	if submit {
```

Comment:
I know we use the global for other reasons, but for this case, does it make sense to call the value directly from flags?

It probably doesn't make a difference, but I'm wondering if it would be more clear here?

Reply:
Also I know you didn't change this

Reply:
I could localise submit with `submit := submit` and not affect the package global. But I'm actually thinking that I do want to change the package global.